components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -393,6 +408,12 @@ - (UIColor *)messageTextColor {
   return self.header.messageTextColor;
 }
 
+- (void)setHeaderDividerColor:(UIColor *)headerDividerColor {
+  _headerDividerColor = [headerDividerColor copy];
```

Comment:
Tip: If the `headerDividerView` is `nonnull`, can this just be a pass-through property?

Reply:
Whoops, forgot to clean this up after addressing other things, thanks for catching this!